.github/workflows/safe-apps-e2e.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        containers: [1, 2, 3]
+        containers: [1, 2]
```

Comment:
Another arbitrary change, what's the reasoning here?

Reply:
@katspaugh there are not many tests in safe apps, that's why reducing number of machines makes sense. As you said, there's more overhead starting the machine than running the tests themselves.